images/plbase/python-requirements.txt

```diff
@@ -11,14 +11,14 @@ isort==5.12.0
 lxml==4.9.3
 lxml-stubs==0.4.0
 matplotlib==3.8.0
-networkx==3.1
+networkx==3.2
 nltk==3.8.1
-numpy==1.26.0
+numpy==1.26.1
 openpyxl==3.1.2
 pandas-stubs==2.1.1.230928
-pandas==2.1.1
+pandas==2.1.2
```

Comment:
Shouldn't do this yet, should be kept in sync with pandas stubs.

Reply:
Gotcha, will revert this.

Reply:
Reverted in a50a696.